config/profile/rhdh/plugin-infra/orchestrator/infra-serverless.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-serverless
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: serverless-operator-group
+  namespace: openshift-serverless
+spec:
+---
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
from RBAC perspective, where do we define the need to create this CR?

Reply:
We do not manage RBAC for infra, this script run by user (which has to have appropriate permissions) directly.